sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaWriter.java

```diff
@@ -50,6 +50,7 @@ public void setup() {
   }
 
   @ProcessElement
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
// TODO : rangadi : explain why this is ok.

Reply:
As above - can I include this in a `Review future handling in KafkaIO` Jira instead please?

Reply:
Yes, either jira, or better, you can add a comment. Reason it is ok to ignore here is that `SendCallback()` keeps track of failures, which are checked inside `finishBundle()`.